src/components/JiscCard/index.jsx

```diff
@@ -60,6 +72,7 @@ JiscCard.propTypes = {
     altTitle: PropTypes.string,
     exampleText: PropTypes.string,
     buttonText: PropTypes.string,
+    clickable: PropTypes.bool,
```

Comment:
Is this prop used?

Reply:
Good spot. This is a legacy prop. I'll remove.